HipercowApi/Tools/Utils.cs

```diff
@@ -30,11 +37,87 @@ public static string HPCString(StoreProperty sp)
         /// Wrapper around Microsoft.Hpc.Scheduler.Properties; return an int if
         /// possible. This is guaranteed to be an int for all valid usage.
         /// </summary>
-        /// <param name="sp">A Microsoft.Hpc.Scheduler.Properties.StoreProperty, returned when querying the cluster.</param>
+        /// <param name="sp">
+        /// A Microsoft.Hpc.Scheduler.Properties.StoreProperty, returned when
+        /// querying the cluster.
+        /// </param>
         /// <returns>An integer value for the property.</returns>
         public static int HPCInt(StoreProperty sp)
         {
             return int.Parse(HPCString(sp));
         }
+
+        /// <summary>
+        /// Helper to return a search filter, which when used queries for all nodes
+        /// except the head node.
+        /// </summary>
+        /// <param name="cluster">The cluster (headnode) name.</param>
+        /// <returns>A FilterCollection object used for filtering.</returns>
+        public static FilterCollection GetFilterNonComputeNodes(
```

Comment:
Ah, so this filters *out* non-compute nodes. This confused me a bit because I thought it was going to filter to *only include* non-compute nodes! This name might be clearer albeit even more verbose... 
```suggestion
        public static FilterCollection GetExcludeNonComputeNodesFilter(
```

Reply:
Fine with me!